app/views/parent_interface/consent_forms/confirm.html.erb

```diff
@@ -29,10 +29,13 @@ end %>
         visually_hidden_text: "consent")
     end
 
+    reason_for_refusal_text = @consent_form.human_enum_name(:reason) +
+                                (@consent_form.reason_notes.present? ? "<br>" + @consent_form.reason_notes : "")
```

Comment:
Feels like an intermediary variable could be used to tidy this up:

```suggestion
    reason_notes = @consent_form.reason_notes.present? ? "<br>" + @consent_form.reason_notes : ""
    reason_for_refusal_text = @consent_form.human_enum_name(:reason) + reason_notes
```

Or did you try this and did it get linted to what you have here? 🙃

Reply:
No, good shout.. fixed.